Documentation/Configuration/ImageRendering/Index.rst

```diff
@@ -0,0 +1,271 @@
+               multiplier {
+                   default = 0.5
+                   large = 0.5
+                   medium = 0.5
+               }
+               gutters {
+                   default = 24
+                   large = 24
+                   medium = 24
+               }
+               corrections {
+                   default = 25
+                   large = 25
+                   medium = 25
+                   small = 50
+                   extrasmall = 50
+               }
+           }
+       }
+   }
+
+
+Configuration options explained
+-------------------------------
+
+Options for variants:
+
++---------------+-----------+-------------------------------------------------+
+| Property      | Data Type | Description                                     |
++===============+===========+=================================================+
+| breakpoint    | integer   | Defines the minimal width in pixels of the      |
+|               |           | device                                          |
++---------------+-----------+-------------------------------------------------+
+| width         | integer   | Defines the effective width in pixels for       |
+|               |           | images for this variant                         |
++---------------+-----------+-------------------------------------------------+
+| aspectRatio   | float     | Optional, defines the default aspect ratio      |
+|               |           | which will override all previously defined      |
+|               |           | ratios on image level                           | 
++---------------+-----------+-------------------------------------------------+
+| sizes         | array     | Optional, defines the available high resolution |
+|               |           | image multiplier for a minimal device pixel     |
+|               |           | ratio per variant                               |  
++---------------+-----------+-------------------------------------------------+
+
+
+Options for backend layouts and content elements:
+
++---------------+-----------+-------------------------------------------------+
+| Property      | Data Type | Description                                     |
++===============+===========+=================================================+
+| multiplier    | integer   | The width is multiplied with this value         |
++---------------+-----------+-------------------------------------------------+
+| gutters       | integer   | This value is added to the width before         |
+|               |           | applying the multiplier and substracted         |
+|               |           | afterwards                                      |
++---------------+-----------+-------------------------------------------------+
+| corrections   | integer   | This value is added as last step to the width   |
+|               |           | after all other calculations                    |
```

Comment:
Why do I need this?
What are the changes in the FE if I use all those options?

Reply:
This explanation and examples are added to the docu now